src/script/common/c_content.h

```diff
@@ -71,137 +71,104 @@ struct collisionMoveResult;
 
 extern struct EnumString es_TileAnimationType[];
 
-void               read_content_features     (lua_State *L, ContentFeatures &f,
-                                              int index);
-void               push_content_features     (lua_State *L,
-                                              const ContentFeatures &c);
+void    read_content_features    (lua_State *L, ContentFeatures &f,int index);
+void    push_content_features    (lua_State *L,const ContentFeatures &c);
 
-void               push_nodebox              (lua_State *L,
-                                              const NodeBox &box);
-void               push_box                  (lua_State *L,
-                                              const std::vector<aabb3f> &box);
+void    push_nodebox             (lua_State *L, const NodeBox &box);
+void    push_box                 (lua_State *L, const std::vector<aabb3f> &box);
 
-void               push_palette              (lua_State *L,
-                                              const std::vector<video::SColor> *palette);
+void    push_palette             (lua_State *L, const std::vector<video::SColor> *palette);
 
-TileDef            read_tiledef              (lua_State *L, int index,
-                                              u8 drawtype, bool special);
+TileDef read_tiledef             (lua_State *L, int index, u8 drawtype, bool special);
 
-void               read_soundspec            (lua_State *L, int index,
-                                              SimpleSoundSpec &spec);
-NodeBox            read_nodebox              (lua_State *L, int index);
+void    read_soundspec           (lua_State *L, int index, SimpleSoundSpec &spec);
+NodeBox read_nodebox             (lua_State *L, int index);
 
-void               read_server_sound_params  (lua_State *L, int index,
-                                              ServerPlayingSound &params);
+void    read_server_sound_params (lua_State *L, int index, ServerPlayingSound &params);
 
-void               push_dig_params           (lua_State *L,
-                                              const DigParams &params);
-void               push_hit_params           (lua_State *L,
-                                              const HitParams &params);
+void    push_dig_params          (lua_State *L, const DigParams &params);
+void    push_hit_params          (lua_State *L, const HitParams &params);
```

Comment:
are tabs being used for aligment here or what are these changes?

Reply:
There were no tabs in the previous code. The alignment was okay and does IMO not need any fixing.

Reply:
@Bituvo please take care of the review so we can merge it

Reply:
I'll revert the alignment changes I made. I may or may not move those to a separate PR